Makefile

```diff
@@ -986,21 +986,17 @@ protos/all: protos/build protos/lint protos/format
 .PHONY: protos/build
 protos/build: buf/installed
 	$(BUF) build
-	cd lib/teleterm && $(BUF) build
-	cd lib/prehog && $(BUF) build
 
 .PHONY: protos/format
 protos/format: buf/installed
 	$(BUF) format -w
-	cd lib/teleterm && $(BUF) format -w
-	cd lib/prehog && $(BUF) format -w
 
 .PHONY: protos/lint
 protos/lint: buf/installed
-	$(BUF) lint
-	cd api/proto && $(BUF) lint --config=buf-legacy.yaml
-	cd lib/teleterm && $(BUF) lint
-	cd lib/prehog && $(BUF) lint
+	$(BUF) lint proto
+	$(BUF) lint --config=api/proto/buf-legacy.yaml api/proto
+	$(BUF) lint lib/teleterm/api/proto
+	$(BUF) lint lib/prehog/proto
```

Comment:
I couldn't find a way to provide three different [inputs](https://docs.buf.build/reference/inputs) to the same `buf` command. So since we need a custom config for a single module, I just repeated the call three times.

Reply:
I think just this is enough, ie, only api/proto needs a second pass:

```
	$(BUF) lint
	$(BUF) lint api/proto --config=api/proto/buf-legacy.yaml
```

The first lint catches all modules, including api/proto/. api/ has a bunch of exemptions in its default buf.yaml, so we do a second pass using buf-legacy.yaml.


(Also, I think you can pass multiple --path flags to buf lint to cover multiple paths.)